controlplane/api/local/connection/connection_helpers.go

```diff
@@ -137,3 +137,11 @@ func (c *Connection) IsComplete() error {
 
 	return nil
 }
+
+func (c *Connection) GetSignature() string {
```

Comment:
As I know JWT has 3 parts:
Header
Payload
Signature

Is this method should return part of JWT or can we rename it?